doc/radosgw/multisite.rst

```diff
@@ -297,7 +297,9 @@ Delete Default Zonegroup and Zone
 
 #. Delete the ``default`` pools in your Ceph storage cluster if they exist.
 
-   .. important:: The following step assumes a multi-site configuration that uses newly installed systems that aren’t currently storing data. DO NOT DELETE the ``default`` zonegroup if you are already using it to store data.
+   .. important:: The following step assumes a multi-site configuration that
+      uses freshly installed systems that store no data. DO NOT DELETE the
```

Comment:
I think we should clarify "systems" here, does it mean hosts? A cluster? A set of pools aka a zone?

Reply:
How does this strike your ear?

``freshly installed pools (which together constitute a "zone")``